### DISCUSSION TITLE
[FEATURE]: gradient edge color

### DISCUSSION THREAD

**Discussion by roonie007** [bcakmakoglu/vue-flow#221] [Ideas]
is it possible to add a gradient colored options line like in the picture below (line 1 -> 2 )

![image](https://user-images.githubusercontent.com/7860859/177408748-b1d9d9c7-2e01-44e0-9d46-8ba97a4d2284.png)

this feature is available in https://github.com/anseki/leader-line, it may helps you

**Comment by bcakmakoglu** [comment]
No, there is no built-in feature to apply a gradient color to an edge, sorry.

You'd have to implement your own custom edge using maybe the library you linked.
[Here's another article demonstrating how to achieve the effect.](https://medium.com/@cereallarceny/getting-gradients-to-follow-along-svg-paths-in-javascript-45357b60bed7)
